FV3/atmos_cubed_sphere/tools/fv_nudge.F90

```diff
@@ -471,11 +472,8 @@ subroutine fv_nwp_nudge ( Time, dt, npx, npy, npz, ps_dt, u_dt, v_dt, t_dt, q_dt
   allocate (ps_obs(is:ie,js:je) )
   allocate ( t_obs(is:ie,js:je,npz) )
   allocate ( q_obs(is:ie,js:je,npz) )
-
-  if ( nudge_winds ) then
-       allocate (u_obs(is:ie,js:je,npz) )
-       allocate (v_obs(is:ie,js:je,npz) )
-  endif
+  allocate (u_obs(is:ie,js:je,npz) )
```

Comment:
Indeed since `get_obs` uses these regardless of what `nudge_winds` is set to it makes sense to do this.  I guess we were always running with `nudge_winds = .true.` before so we never ran into any problems.

Reply:
Yes exactly. Without this change (and the one within the `nudge_debug` if statement) the model seg faults with `nudge_winds = .false.`.